database/scripts/20220110_234447_modificar_valor_metadato_up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE acta_recibido.campo
+    ADD COLUMN metadato jsonb;
+
+ALTER TABLE acta_recibido.elemento_campo
+    DROP COLUMN IF EXISTS valor CASCADE;
+
```

Comment:
Dejar comentado, advertir, que el SQL implica perdida de datos:

```suggestion
-- ADVERTENCIA: Las 2 sentencias SQL siguientes implican
-- una POSIBLE PÉRDIDA DE DATOS.
-- Se recomienda mejorar las dos líneas siguientes para que en
-- vez de simplemente eliminar y volver a agregar la columna,
-- se transformen los valores que estaban antes
ALTER TABLE acta_recibido.elemento_campo
    DROP COLUMN IF EXISTS valor CASCADE;
```